doc/tutorials/acrn_on_qemu.rst

```diff
@@ -0,0 +1,292 @@
+      [    2.368358] ACRN HVLog: Initialized hvlog module with 4 cpu
+      [    2.727905] systemd[1]: Set hostname to <ACRNSOS>.
+
+8. When shutting down, make sure to cleanly destroy the Service VM to prevent crashes in subsequent boots. This can be done using,
+
+   .. code-block:: none
+
+      virsh destroy ACRNSOS # where ACRNSOS is the virsh domain name.
+
+
+Service VM Networking updates for User VM
+*****************************************
+Follow these steps to enable networking for the User VM (L2 guest):
+
+1. Edit your :file:`/etc/netplan/01-netcfg.yaml` file to add acrn-br0 as below,
+
+   .. code-block:: none
+
+      network:
+         version: 2
+         renderer: networkd
+         ethernets:
+            enp1s0:
+               dhcp4: no
+         bridges:
+            acrn-br0:
+               interfaces: [enp1s0]
+               dhcp4: true
+               dhcp6: no
+
+2. Apply the new network configuration by,
+
+   .. code-block:: none
+
+      $ cd /etc/netplan
+      $ sudo netplan generate
+      $ sudo netplan apply
+
+3. Create a tap interface (tap0) and add the tap interface as part of the acrn-br0 using the below steps,
+
+   a. Copy files ``misc/acrnbridge/acrn.network`` and ``misc/acrnbridge/tap0.netdev`` from the cloned ACRN repo to :file:`/usr/lib/system/network`.
+   b. Rename ``acrn.network`` to ``50-acrn.network``.
+   c. Rename ``tap0.netdev`` to ``50-tap0.netdev``.
+
+4. Restart ACRNSOS guest (L1 guest) to complete the setup and start with bring-up of User VM
+
+
+Bring-up User VM (L2 Guest)
+***************************
+1. Build the device-model, using ``make devicemodel`` and copy acrn-dm to ACRNSOS guest (L1 guest) directory ``/usr/bin/acrn-dm``
+
+   .. note::
+      It should be already built as part of :ref:`install_acrn_hypervisor`.
+
+2. On the ACRNSOS guest, install shared libraries for acrn-dm (if not already installed).
+
+   .. code-block:: none
+
+      $ sudo apt-get install libpciaccess-dev
+
+3. Install latest `IASL tool <https://acpica.org/downloads>`_ and copy the binary to ``/usr/sbin/iasl``.
+   For this setup, used IASL 20200326 version but anything after 20190215 should be good.
+
+4. Clone latest stable version or master and build ACRN User VM Kernel.
+
+   .. code-block:: none
+
+      $ git clone https://github.com/projectacrn/acrn-kernel
+      $ cd acrn-kernel
+      $ cp kernel_config_uos to .config
+      $ make
+
+   Once the User VM kernel is built successfully, copy ``arch/x86/boot/bzImage`` to  ACRNSOS (L1 guest) and rename this to ``bzImage_uos``. Need this to launch the User VM (L2 guest)
+
+   .. note::
+      The User VM kernel contains all needed drivers so you won't need to install extra kernel modules.
+
+5. Build ubuntu.img using :ref:`build-the-ubuntu-kvm-image` and copy it to the ACRNSOS (L1 Guest).
+   Alternatively you can also use virt-install to create a User VM image similar to ACRNSOS as shown below,
+
+   .. code-block:: none
+
+      virt-install \
```

Comment:
```suggestion
      $ virt-install \
```

Reply:
Do you need `sudo` to run `virt-install`?

Reply:
sudo is not required.